<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="16" hostname="DESKTOP-K0SFL7M" failures="1" tests="1" name="Default test" time="243.235" errors="0" timestamp="2021-07-02T16:28:35 PKT">
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="RunAPI" time="224.966">
    <failure type="java.lang.Exception" message="java.lang.AssertionError: expected [12] but found [10]">
      <![CDATA[java.lang.Exception: java.lang.AssertionError: expected [12] but found [10]
at Test.Ancera.Reports.CoccidiaLog.RunAPI(CoccidiaLog.java:357)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: expected [12] but found [10]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Test.Ancera.Reports.CoccidiaLog.RunAPI(CoccidiaLog.java:349)
... 28 more
]]>
    </failure>
  </testcase> <!-- RunAPI -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="TemplateExport" time="0.0">
    <ignored/>
  </testcase> <!-- TemplateExport -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="Pagination" time="0.0">
    <ignored/>
  </testcase> <!-- Pagination -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="FieldAccessView" time="0.0">
    <ignored/>
  </testcase> <!-- FieldAccessView -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="FieldAccessUnview" time="0.0">
    <ignored/>
  </testcase> <!-- FieldAccessUnview -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="RowsPerPage" time="0.0">
    <ignored/>
  </testcase> <!-- RowsPerPage -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="TestFilters" time="0.0">
    <ignored/>
  </testcase> <!-- TestFilters -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="NavigateCoccidia" time="0.0">
    <ignored/>
  </testcase> <!-- NavigateCoccidia -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="DateFilter" time="0.0">
    <ignored/>
  </testcase> <!-- DateFilter -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="CoccidiaLock" time="0.0">
    <ignored/>
  </testcase> <!-- CoccidiaLock -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="CSVAuditExport" time="0.0">
    <ignored/>
  </testcase> <!-- CSVAuditExport -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="PNGExport" time="0.0">
    <ignored/>
  </testcase> <!-- PNGExport -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="DateLockFilter" time="0.0">
    <ignored/>
  </testcase> <!-- DateLockFilter -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="TestFilter" time="0.0">
    <ignored/>
  </testcase> <!-- TestFilter -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="CSVExport" time="0.0">
    <ignored/>
  </testcase> <!-- CSVExport -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="Sorting" time="0.0">
    <ignored/>
  </testcase> <!-- Sorting -->
  <testcase classname="Test.Ancera.Reports.CoccidiaLog" name="wildcard" time="0.0">
    <ignored/>
  </testcase> <!-- wildcard -->
</testsuite> <!-- Default test -->
